src/System.Private.CoreLib/shared/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Float.cs

```diff
@@ -61,32 +61,17 @@ public static bool TryFormat(float value, Span<byte> destination, out int bytesW
 
         private static bool TryFormatFloatingPoint<T>(T value, Span<byte> destination, out int bytesWritten, StandardFormat format) where T : IFormattable, ISpanFormattable
         {
+            Span<char> formatText = stackalloc char[StandardFormat.FormatStringLength];
```

Comment:
Can we avoid the unnecessary stackalloc for the default case?

Would @stephentoub's suggestion work?
```C#
Span<char> formatText = stackalloc char[0];
```

Reply:
Yes, I've updated accordingly.

I'm still not convinced this was worth the effort, considering it is 6-bytes we are stack-allocating 😄